app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.kt

```diff
@@ -326,7 +326,7 @@ class FeedFragment : BaseStateFragment<FeedState>() {
         if (context == null || context.resources == null || activity == null) return
 
         val entries = ArrayList<StreamDialogEntry>()
-        if (PlayerHolder.getInstance().getType() != null) {
+        if (PlayerHolder.getInstance().isPlayerOpen) {
```

Comment:
This doesn't build. Please test on your machine before pushing here next time ;-)

Reply:
weird
I've build it containing that line multiple times on my machine, including just now.

Going to change it anyways if you had problems with it.
Just something to note.